### PR TITLE
Add capa rule load-code/shellcode/execute-shellcode-via-callback-functions.yml

### PR DESCRIPTION
**Pull request by ejfocampo**

### CODE REVIEW

**Review by mike-hunhoff — CHANGES_REQUESTED** [review]
Thank you @ejfocampo, looks great! I suggested minor changes then LGTM 🚀

File: load-code/shellcode/execute-shellcode-via-callback-functions.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - https://www.trendmicro.com/en_us/research/22/k/earth-preta-spear-phishing-governments-worldwide.html
+    examples:
+      - 10cd7afd580ee9c222b0a87ff241d306:0x10008BE0
+      - 268d61837aa248c1d49a973612a129ce:0x1000CEC0
+      - 4a2992b4c7a1573bf7c74065e3bf5b0d:0x1000D050
+  features:
+    - and:
+      - match: allocate RWX memory
+      - or:
+        - api: EnumDateFormatsA
```
Comment (mike-hunhoff) [inline load-code/shellcode/execute-shellcode-via-callback-functions.yml:22]: capa automatically handles Windows API functions ending in `A` and `W` so we can drop the `A` here and capa will detect both forms. Apply this to the APIs listed below.

Comment (ejfocampo) [reply load-code/shellcode/execute-shellcode-via-callback-functions.yml]: Omitted A and W on relevant APIs as suggested. Thanks for the tip @mike-hunhoff!

File: load-code/shellcode/execute-shellcode-via-callback-functions.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        - api: EnumChildWindows
+        - api: EnumDesktopsW
+        - api: EnumDesktopWindows
+        - api: EnumSystemCodePagesA
+        - api: EnumSystemCodePagesW
+        - api: EnumSystemGeoID
+        - api: EnumSystemLanguageGroupsA
+        - api: EnumSystemLocalesA
+        - api: EnumThreadWindows
+        - api: EnumUILanguagesA
```
[the remark this answers is not in the document]
Comment (ejfocampo) [reply load-code/shellcode/execute-shellcode-via-callback-functions.yml]: I talked to the author of the rule - Jakub, and got the confirmation that we can merge the rules. Any guidance on how to proceed? Thanks

[the remark this answers is not in the document]
Comment (ejfocampo) [reply load-code/shellcode/execute-shellcode-via-callback-functions.yml]: Merged the two rules and deleted execute-shellcode-via-enumuilanguages.yml. Thanks

**Review by mike-hunhoff — APPROVED** [review]
looks great @ejfocampo , LGTM 🚀

**Pull request merged**